python/ml4ir/applications/classification/pipeline.py

```diff
@@ -164,6 +166,102 @@ def get_relevance_dataset(
 
         return relevance_dataset
 
+    def evaluate(
+        self,
+        test_dataset: data.TFRecordDataset,
+        inference_signature: str = None,
+        additional_features: dict = {},
+        group_metrics_min_queries: int = 50,
+        logs_dir: Optional[str] = None,
+        logging_frequency: int = 25,
+    ):
```

Comment:
I kept the same signature with the ranking and the base version of the function.